features/comments/commenting.feature

```diff
@@ -27,6 +27,34 @@ Feature: Commenting
     And I should see "Hello from Comment"
     And I should see a comment by "admin@example.com"
 
+  Scenario: Create an anonymous comment without authentication
+    Given a show configuration of:
+      """
+        # without :required options for rails 5
+        if Rails::VERSION::MAJOR >= 5
+          # assume Rails.application.config.active_record.belongs_to_required_by_default = true
+          ActiveAdmin::Comment.belongs_to_required_by_default = true
+          ActiveAdmin::Comment.belongs_to :author, polymorphic: true
+        else # use required: true for rails 4
+          ActiveAdmin::Comment.belongs_to :author, polymorphic: true, required: true
+        end
```

Comment:
Just a question, no need to change anything in the PR. If we enable `belongs_to_required_by_default = true` in our test app (which we definitely should), would this whole block be needed for the test?

Reply:
no need, but I think there are many tests need to be fixed with `belongs_to_required_by_default = true`

Reply:
I merged #5495. If you'd like to wrap this up, please rebase and remove the whole block above!

Reply:
nice work, I will update this PR.

Reply:
Thanks @rainchen!